repo-update-cadist

```diff
@@ -128,39 +144,23 @@ for TYPES in NEW IGTFNEW; do
     rm -rf "$DOWNLOADDIR"
 
 
-    ## Download the "version" file from SVN - this has a name like
+    ## Generate the "version" file (aka manifest) - this has a name like
     ## ca-certs-version-1.59NEW and is a txt file with the checksum of the
     ## tarball in it. Then rename it to add the "-new" or "-igtf-new"
     ## suffix.
-    VERSIONFILE_URL=${CADISTREPO}/${CADISTREPORELEASETYPE}/ca-certs-version-${VERSION_CA}${SUFFIX}
     VERSIONFILE=${TMPROOT}/cadist/ca-certs-version${FILEEXT}
-    if ! svn export -q --force "$VERSIONFILE_URL" "$VERSIONFILE"; then
-        message "$VERSIONFILE: unable to download"
-        message "Upstream URL: $VERSIONFILE_URL"
-        exit 1
-    fi
-
-    ## Check the checksums
-    if fgrep -q tarball_sha256sum "$VERSIONFILE"; then
-        expected_sum=$(
-            perl -lne '/^\s*tarball_sha256sum\s*=\s*(\w+)/ and print "$1"' \
-                "$VERSIONFILE")
-        actual_sum=$(sha256sum "$CATARBALL" | awk '{print $1}')
-    elif fgrep -q tarball_md5sum "$VERSIONFILE"; then
-        expected_sum=$(
-            perl -lne '/^\s*tarball_md5sum\s*=\s*(\w+)/ and print "$1"' \
-                "$VERSIONFILE")
-        actual_sum=$(md5sum "$CATARBALL" | awk '{print $1}')
-    else
-        message "$CATARBALL: checksum not found in version file $VERSIONFILE"
-        exit 1
-    fi
-    if [[ $expected_sum != $actual_sum ]]; then
-        message "$CATARBALL: checksum mismatch"
-        message "Expected: $expected_sum"
-        message "Actual:   $actual_sum"
-        exit 1
-    fi
+    REPO_CADIST=http://repo.opensciencegrid.org/pacman/cadist
```

Comment:
Please drop pacman from this line: http://repo.opensciencegrid.org/cadist

Reply:
Updated!  I was trying to reproduce the manifest format seen in the [current manifest file](http://repo.opensciencegrid.org/cadist/ca-certs-version-igtf-new) ... but yeah looks like the leading pacman dir is unnecessary now.